software/base/src/test/java/org/apache/brooklyn/entity/software/base/test/location/WinRmMachineLocationLiveTest.java

```diff
@@ -65,7 +65,7 @@ public void setUpClass() throws Exception {
         JcloudsLocation loc = (JcloudsLocation) mgmt.getLocationRegistry().resolve("jclouds:aws-ec2:us-west-2", ImmutableMap.of(
                 "inboundPorts", ImmutableList.of(5985, 3389),
                 "displayName", "AWS Oregon (Windows)",
-                "imageId", "us-west-2/ami-8fd3f9bf",
+                "imageNameRegex", ".*Windows.*2012.*",
```

Comment:
A customer said that they also needed to specify an image owner, in order to avoid a marketplace windows AMI being picked. Can you search for AMIs that match that regex to see if that is a risk?

For example:

    brooklyn.location.named.mywindows1.imageOwner=801119661308
    brooklyn.location.named.mywindows1.imageNameRegex=Windows_Server-2012-R2_RTM-English-64Bit-Base-.*

This is used within our jclouds integration code (in ComputeServiceRegistryImpl.findComputeService) where we have:

   properties.setProperty(PROPERTY_EC2_AMI_QUERY,
   "owner-id="+conf.getStringKey("imageOwner")+";state=available;image-type=machine");

(It's a bit ugly having the code there; would be nicer if that were a first-class property on jclouds' AWSEC2TemplateOptions. That's an improvement we could make in jclouds. It would also be nice to have a config key for `imageOwner` rather than just this hard-coded constant being looked up in the config!).